### ISSUE TITLE
想问一下 针对PET的合成

### ISSUE THREAD

**Issue by MaoFuyou** [Yaziwel/Restore-RWKV#11]
您好 作者，想问一下 如果做到对nii.gz 数据进行切片处理的呢？是有相关集成好的代码还是用的相关软件？谢谢

**Comment by Yaziwel** [comment]
> 您好 作者，想问一下 如果做到对nii.gz 数据进行切片处理的呢？是有相关集成好的代码还是用的相关软件？谢谢

优点没太懂，是你自己的数据吗？nii数据或者nii.gz可以使用SimpleITK库读写，切片自己提取就行

**Comment by MaoFuyou** [comment]
对 我看您的all-in-one数据里面；涉及到了.bin文件 这个文件也是切片后的么，还有如果是单个切片的话 是以nii保存是吧 就没必要nii.gz了是吧

**Comment by Yaziwel** [comment]
gz是进行了压缩，可以不用gz，不管2D还是3D都可以存nii或者nii.gz



---Original---
From: "Mao ***@***.***&gt;
Sent at: 2024年9月3日(Tue) PM5:27
To: ***@***.***&gt;;
Cc: "Zhiwen ***@***.******@***.***&gt;;
Subject: Re: [Yaziwel/Restore-RWKV] 想问一下 针对PET的合成 (Issue #11)




 
对 我看您的all-in-one数据里面；涉及到了.bin文件 这个文件也是切片后的么，还有如果是单个切片的话 是以nii保存是吧 就没必要nii.gz了是吧

**Comment by MaoFuyou** [comment]
![Image](https://github.com/user-attachments/assets/192c131a-639d-4478-9e32-3928ec9a6fdd)想问一下这是怎么回事 本身的 Restore_RWKV.py的测试已经通过了 但是 我自己写的train就不行了
